build.gradle.kts

```diff
@@ -582,9 +616,38 @@ tasks {
     destinationDirectory.set(projectDir.dir("distribution"))
   }
 
+  val assembleIcebergRESTServer by registering(Tar::class) {
+    dependsOn("compileIcebergRESTServer")
+    // fix gradlew assembleDistribution error
```

Comment:
What's the problem here?

Reply:
Gradlew reports the task dependency error previously, but after some change, I could run pass it with removing the `runAfter`,  so remove it now.